src/bigquery.rs

```diff
@@ -179,10 +182,22 @@ impl TranslateFrom<ast::Tag> for Tag {
         } else {
             Mode::Required
         };
+
+        // If schema uses both title and description, format description as "<title> - <description>".
+        // Otherwise set description to whatever is available.
+        let description = match tag.title {
+            Some(title) => match tag.description {
+                Some(description) => Some(format!("{} - {}", title, description)),
```

Comment:
One thing to mention here is that when deserializing BigQuery schemas into `ast::Tag`s then currently there is no logic implemented that tries to split the description into `title` and `description`. So if the description in BigQuery schema is something like "TITLE - DESCRIPTION" then for `ast::Tag` the `title` attribute will remain `None` and description will be set to the description.

Not sure if that is an issue here? It would mean that transpiling a schema to its BigQuery schema and transpiling it back might result in a different schema.

Reply:
It's not an issue since there are other ambiguities that are more fundamental than the descriptions, with `snake_casing` being the largest that comes to mind. Because of this, the transpiler has only been developed to run in one direction. There hasn't been any effort to normalize schemas via this tool.